.github/workflows/main.yml

```diff
@@ -103,7 +105,8 @@ jobs:
           bazel test --show_timestamps       \
             //software:unix_full_system           \
             //software/simulated_tests/...   \
-            //software/ai/hl/...
+            //software/ai/hl/... \
+            //software/ai/navigator/...
```

Comment:
you need to make this change one more time for "Simulated Tests 3"

Reply:
done